plugins/base/src/main/resources/dokka/scripts/platform-content-handler.js

```diff
@@ -261,13 +257,24 @@ function toggleSections(target) {
             }
         }
     }
-
+    const toggleTargets = target.getAttribute("data-togglable").split(",")
+    const activateTabsBody = (containerClass) => {
+        document.querySelectorAll("." + containerClass + " *[data-togglable]")
+            .forEach(child => {
+                    if (toggleTargets.includes(child.getAttribute("data-togglable"))) {
+                        child.setAttribute("data-active", "")
+                    } else if(!child.classList.contains("sourceset-dependent-content")) {
+                        child.removeAttribute("data-active")
+                    }
```

Comment:
My code wasn't working and I had to debug this piece of code. Couldn't understand why we need to have this second branch for `sourceset-dependent-content` - is there anything special about it?